docs/source/releases/v1_13_0a0.rst

```diff
@@ -336,6 +337,34 @@ implement those changes.
     modified: geoips/plugin_registry.py
     modified: geoips/geoips_utils.py
 
+Refactored ``geoips list interface`` to use a class factory
+-----------------------------------------------------------
+
+*From GEOIPS#576: 2024-05-11, CLI: Reduce number of command layers where possible and appropriate*
+*From GEOIPS#573: 2024-05-11, Use class factories to generate subcommands for commands like ``geoips get family`` and ``geoips get interface``
+
+While this PR doesn't fix all of the problems addressed in the issues above, it is our
+initial attempt at creating a class factory for certain CLI commands to reduce the
+verbage needed to execute those commands. Spefically in this branch, we've addressed the
+command class ``GeoipsListSingleInterface``. We now use the aforementioned class as a
+base class to build ``GeoipsListSingleInterface<interface_name>`` classes at runtime.
+
+By doing so, we can now run ``geoips list <interface_name>`` rather than
```

Comment:
Will this always be unique ? Ie, there will not be similarly named interfaces, plugins,  families that would all come after "geoips list" ? Or would it always be "geoips list <interface name> ..."?

Reply:
I think this will always be unique. I don't think we'll ever support two interfaces that are named the same thing and retrieving plugins is handled at an interface by interface basis, so we only require that plugin names be unique within a given interface. I think that answers your question, but I'm not sure.